source/read/project.txt

```diff
@@ -0,0 +1,126 @@
+.. _kotlin-sync-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/read/project.kt
+   :start-after: start-data-class
+   :end-before: end-data-class
+   :language: kotlin
+   :copyable:
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
+exclusion statements in a single projection, unless you are excluding the
+``_id`` field.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following syntax to specify the fields to include in the result:
+
+.. code-block:: kotlin
+
+   val projection = Projection.fields(
+       Projections.include(<fieldName1>, <fieldName2>, ...),
+   )
+
+The following example uses the ``find()`` method to find all restaurants with
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+return only the ``name``, ``cuisine``, and ``borough`` fields of the returned
+documents.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/project.kt
+      :start-after: start-project
+      :end-before: end-project
+      :language: kotlin
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Restaurant(id=5eb3d668b31de5d588f429e2, name=Emerald Pub, borough=Manhattan, cuisine=American)
+      Restaurant(id=5eb3d668b31de5d588f432dd, name=Emerald Pub, borough=Queens, cuisine=American)
+
+When you use a projection to specify fields to include in the return
+document, the ``_id`` field is also included by default. All other fields are
+implicitly excluded. To remove the ``_id`` field from the return
+document, you must :ref:`explicitly exclude it <kotlin-sync-project-exclude-id>`.
+
+.. _kotlin-sync-project-exclude-id:
+
+Exclude the ``_id`` Field
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying fields to include, you can also exclude the ``_id`` field from
```

Comment:
Can you only exclude the _id field when specifying fields to include, or can you exclude just the _id field without using any includes statements?